backend/utils/llm.py

```diff
@@ -40,6 +40,30 @@ def get_response(system_prompt, user_prompt, tools=None) -> ChatCompletionRespon
         ],
         tools=tools,
         tool_choice=tool_choice,
+        temperature=0
     )
-    logger.info('{0} response : "{1}"'.format(config.mistral_model, response.choices[0].message.content))
+    logger.debug('{0} response : "{1}"'.format(config.mistral_model, response.choices[0].message.content))
     return response
+
+
+# TODO: Refactor - 1 get_response method and 1 call_model method
```

Comment:
hmm yeah I wonder whether we need some sort of `message_builder` to create the messages, with how ever many system prompts needed, and then could just pass `messages` into `get_response`. Is your intention to do all these refactors in the wire up stage?

Reply:
Or perhaps even later. I just want to get to the PoC point as soon as possible now, knowing we may scrap parts of it anyway. A bad attitude to writing code but potentially an appropriate one here as we're still not 100% on the feasibility of this approach overall

Reply:
This also extends to the poor quality testing I've done here, all to be bolstered and improved if we keep this approach and (as a result) these files

Reply:
yeah very good points, just me having refactoring ideas, not for now